packages/create-react-admin/src/generateAppFile.ts

```diff
@@ -24,24 +24,21 @@ ${
 export const App = () => (
     <Admin
         layout={Layout}
-        ${
-            state.dataProvider !== 'none'
-                ? `dataProvider={dataProvider}\n\t`
-                : ''
-        }${
+        ${state.dataProvider !== 'none' ? `dataProvider={dataProvider}` : ''}${
             state.authProvider !== 'none'
-                ? `\tauthProvider={authProvider}\n\t`
+                ? `\n\tauthProvider={authProvider}`
                 : ''
-        }>
+        }
+    >
         ${state.resources
             .map(
                 resource =>
                     `<Resource name="${resource}" list={ListGuesser} edit={EditGuesser} show={ShowGuesser} />`
             )
-            .join('\n\t\t')}
+            .join('\n\t')}
```

Comment:
this made alignment worse according to my tests. The root issue was the use of tabs instead of spaces I believe...